src/Samples/SubModelSeq/Program.fs

```diff
@@ -13,177 +31,199 @@ module List =
         | a when a = j -> i
         | a -> a)
 
+  let swapWithNext i = swap i (i + 1)
+  let swapWithPrev i = swap i (i - 1)
+
+  let cons a ma = a :: ma
 
+        
 [<AutoOpen>]
-module Domain =
+module IdentifiableTypes =
+
+  type Identifiable<'a> =
+    { Id: Guid
+      Value: 'a }
+  
+  module Identifiable =
+    module Id =
+      let get m = m.Id
+      let set v m = { m with Id = v }
+      let map x = x |> map get set
+    module Value =
+      let get m = m.Value
+      let set v m = { m with Value = v }
+      let map f = f |> map get set
+      let mapIf p f = f |> map |> mapIf p
+      let mapIfId id f = f |> mapIf (fun m -> m.Id = id)
+  
+    let create v id =
+      { Id = id
+        Value = v }
```

Comment:
I don't think this function adds anything. It's just as easy to create the type without it.

```suggestion
```

Reply:
Sure.  I made this change.